Framework/Kernel/src/Matrix.cpp

```diff
@@ -156,63 +160,61 @@ Matrix<T>::Matrix(const std::vector<T> &data, const size_t nrow,
   setMem(nrow, ncol);
 
   size_t ic(0);
-  for (size_t i = 0; i < nx; i++) {
-    for (size_t j = 0; j < ny; j++) {
-      V[i][j] = data[ic];
+  for (size_t i = 0; i < m_numRowsX; i++) {
+    for (size_t j = 0; j < m_numColumnsY; j++) {
+      m_rawData[i][j] = data[ic];
       ic++;
     }
   }
 }
 
 template <typename T>
 Matrix<T>::Matrix(const Matrix<T> &A, const size_t nrow, const size_t ncol)
-    : nx(A.nx - 1), ny(A.ny - 1), V(nullptr)
+    : m_numRowsX(A.m_numRowsX - 1), m_numColumnsY(A.m_numColumnsY - 1)
 /**
   Constructor with for a missing row/column.
   @param A :: The input matrix
   @param nrow :: number of row to miss
   @param ncol :: number of column to miss
 */
 {
-  if (nrow > nx)
-    throw Kernel::Exception::IndexError(nrow, A.nx,
+  if (nrow > m_numRowsX)
+    throw Kernel::Exception::IndexError(nrow, A.m_numRowsX,
                                         "Matrix::Constructor without col");
-  if (ncol > ny)
-    throw Kernel::Exception::IndexError(ncol, A.ny,
+  if (ncol > m_numColumnsY)
+    throw Kernel::Exception::IndexError(ncol, A.m_numColumnsY,
                                         "Matrix::Constructor without col");
-  setMem(nx, ny);
-  if (V) {
-    size_t iR(0);
-    for (size_t i = 0; i <= nx; i++) {
-      if (i != nrow) {
-        size_t jR(0);
-        for (size_t j = 0; j <= ny; j++) {
-          if (j != ncol) {
-
-            V[iR][jR] = A.V[i][j];
-            jR++;
-          }
+  setMem(m_numRowsX, m_numColumnsY);
+  size_t iR(0);
+  for (size_t i = 0; i <= m_numRowsX; i++) {
+    if (i != nrow) {
+      size_t jR(0);
+      for (size_t j = 0; j <= m_numColumnsY; j++) {
+        if (j != ncol) {
+
```

Comment:
Really picky I know but as there are other changes can we remove this whitespace?